core/src/main/java/org/apache/iceberg/CatalogProperties.java

```diff
@@ -86,8 +86,8 @@ private CatalogProperties() {}
    * <ul>
    *   <li>Zero - Cache entries expires only if it gets evicted due to memory pressure from {@link
    *       #IO_MANIFEST_CACHE_MAX_TOTAL_BYTES} setting.
-   *   <li>Positive Values - Cache entries expire if not accessed via the cache after this many
-   *       milliseconds
+   *   <li>Positive Values - Cache entries expire after a fixed time period, regardless of whether
+   *        they are accessed via the cache during this time
```

Comment:
I don't think manifest cache is changed, right?

Reply:
Correct, manifest cache use `weakKeys` and `softValue` to expire entries by gc.